.gitignore

```diff
@@ -4,4 +4,7 @@
 
 # Composer
 /vendor
-/composer.lock
\ No newline at end of file
+/composer.lock
+
+# Tests
+/tests/*.compiled.php
```

Comment:
A .gitignore is present in /tests/. If we do this, we will use this separated .gitignore